src/app/integrations/edit-page/step-configure/filter-steps/advanced-filter.component.html

```diff
@@ -0,0 +1,50 @@
+<div class="advanced-filter">
+  <div class="clearfix">
+    <div class="title">
+      <h1>Configure Advanced Filter Step</h1>
+    </div>
+    <p>Fill out the fields associated with the selected step.</p>
+    <form class="form-horizontal">
+      <div>
+        <p>Continue only if incoming data match all of the following:</p>
+      </div>
+      <div class="form-group">
+        <div class="col-xs-9">
+          <input type="text"
+                 name="statusTextInput"
+                 data-id="statusTextInput"
+                 placeholder="{{ name }}"
+                 [(ngModel)]="name"
+                 class="form-control">
+        </div>
+      </div>
+      <div class="form-group"></div>
+      <div class="add-rule">
+        <a href="">+ Add another rule</a>
```

Comment:
Yeah I wasn't even looking at the advanced filter tbh, but yes that will be moved to the basic filter.